README.md

```diff
@@ -207,13 +203,17 @@ permit becomes available.
 
 Each host has a concurrency limiter which protects servers by stopping requests getting out the door on the client-side.
 Permits are decreased after receiving 308 or 501-599 response, or encountering a network error (`IOException`).
-Otherwise, they are increased.
+429 or 500 responses have no change. Otherwise, permits are increased.
+
+Host limits are based on failures that indicate the target host overall is in a degraded state.
```

Comment:
this is an explanation/reason for why the status codes above are handled the way they  are